hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -324,6 +324,10 @@ def get_working_days_details(
 		holidays = self.get_holidays_for_employee(self.start_date, self.end_date)
 
 		if not cint(include_holidays_in_total_working_days):
+			
+			all_date_list = date_range(getdate(self.start_date), getdate(self.end_date))
```

Comment:
There is no date_range function in frappe, rather date_diff exists. If you are using pandas lib then we are not supporting that.

Reply:
Actually it is custom function..added at the end of the file.